docs/source/home.js

```diff
@@ -1,7 +1,3 @@
+const package = require('./../../package.json')
 
-var package = require('./../../package.json')
-
-module.exports = {
-    package: package
-  , title: 'ODM'
-}
+module.exports = { package: package, title: 'ODM' };
```

Comment:
This script doesn't look like it is imported anywhere?

Reply:
seems like it, removed with cef2843ccd55125781b2d1c456e4add963da1211, it was just touched up by this pr because of auto-format

Reply:
while re-doing this PR i noticed that the entry for `home` in `source/index` was lost in translation somewhere